django/contrib/admin/static/admin/css/rtl.css

```diff
@@ -254,3 +258,7 @@ div.colM {
 .submit-row input {
     float: left;
 }
+
+.related-widget-wrapper {
```

Comment:
No, without it, widgets will be floated to the left in RTL languages as already is. That's the whole point of my commit.